src/FileSystemActions.ts

```diff
@@ -0,0 +1,154 @@
+"use strict";
+
+import {
+    window,
+    workspace,
+    Disposable,
+    FileCreateEvent,
+    FileDeleteEvent,
+    TextDocument,
+    TextDocumentChangeEvent,
+    Uri,
+    TextDocumentSaveReason
+} from "vscode";
+
+import * as micromatch from "micromatch";
+
+import { Display } from "./Display";
+import { PerforceCommands } from "./PerforceCommands";
+import { PerforceSCMProvider } from "./ScmProvider";
```

Comment:
now that `parse-gitignore` is not required can you run `npm uninstall parse-gitignore` to remove it from the dependencies :)

Reply:
yup!  forgot about this.